apps/framework-cli/src/infrastructure/olap/clickhouse.rs

```diff
@@ -1767,6 +1935,10 @@ impl OlapOperations for ConfiguredDBClient {
                 indexes,
                 database: Some(database),
                 table_ttl_setting,
+                // cluster_name is always None from introspection because ClickHouse doesn't store
+                // the ON CLUSTER clause - it's only used during DDL execution and isn't persisted
+                // in system tables. Users must manually specify cluster in their table configs.
+                cluster_name: None,
```

Comment:
would it make sense to provide some sort of escape hatch say let the user set an obscure env var in-case the on cluster stuff changes and we were not able to pick up that change etc since its not tied to our normal reconciliation process?

Reply:
Ok cool if the config file provides the hatch that makes sense. I was thinking worst case scenario you can provide an env var they can set that would pick up the change etc, but I think with the config file it fines.